### PR TITLE
Small changes to the database set-up page

### PR DESCRIPTION
**Pull request by victoriaspek**
Wanted to add a link to the troubleshooting guide, made some other small changes as well.

### CODE REVIEW

**Review by salsakran — APPROVED** [review]
+1 to the link, I didn't verify that the images make sense though

**Comment by salsakran** [comment]
one minor point - for docs changes can you use `[ci skip]` on the commit message so that it doesn't tie up testing resources?

File: docs/administration-guide/01-managing-databases.md [hunk 1]
```diff
@@ -75,7 +79,7 @@ Syncing can be set to hourly, or daily at a specific time. Syncing can't be turn
 
 If you'd like to sync your database manually at any time, click on it from the Databases list in the admin panel and click on the Sync button on the right side of the screen:
```
Comment (mazameli) [inline docs/administration-guide/01-managing-databases.md]: Guess the button technically says "Sync database schema now".

**Review by mazameli — APPROVED** [review]
Left one small comments, but seems chill.

**Pull request merged**